### ISSUE TITLE
TorchBench v1 score regresses 2% on 20210605

### ISSUE THREAD

**Issue by xuzhao9** [pytorch/pytorch#66259]
## 🐛 Bug

TorchBench v1.1 score regresses 2% between 20210604(f87aa23125ff92520204fa0ca201c2d169fd3e17) and 20210605 (c769300301e8bc775e54cdd114e2beaa11b1f893).

Affected tests and regression percentage:
```
test_eval[densenet121-cuda-eager] +9.150%
test_train[dlrm-cpu-eager] +8.593%
test_eval[dlrm-cpu-eager] +7.779%
test_train[dlrm-cuda-eager] +7.177%
test_eval[hf_DistilBert-cpu-eager] +18.530%
test_train[maml-cuda-eager] +7.757%
test_eval[maml-cuda-eager] +7.226%
test_train[maml_omniglot-cuda-eager] +9.572%
test_eval[pytorch_struct-cuda-jit] +7.200%
test_eval[pytorch_struct-cuda-eager] +7.668%
test_eval[shufflenet_v2_x1_0-cuda-eager] +8.357%
test_eval[speech_transformer-cuda-eager] +10.350%
test_eval[timm_efficientnet-cuda-jit] +7.052%
test_eval[vgg16-cpu-eager] +15.285%
```

Environment:

- CUDA 10.2
- PyTorch nightly 1.10.0dev20210605+cu102

cc @VitalyFedyunin @ngimel

**Comment by xuzhao9** [comment]
The bisection job returns empty result. So the reason could be some changes in the build environment. After taking a closer look, it seems https://github.com/pytorch/pytorch/commit/c7a3a13baba0d547c5c20579328b0b3d83b94656 is the root cause.

**Comment by xuzhao9** [comment]
Closed as this only happens on CUDA 10.2, which is being deprecated.
